updatecli/updatecli.d/updatek3sroot.yaml

```diff
@@ -0,0 +1,65 @@
+---
+name: "Update k3sroot version" 
+
+sources:
+ k3sroot:
+   name: Get k3s-root version
+   kind: githubrelease
+   spec:
+     owner: rancher
+     repository: k3s-root
+     token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+     typefilter:
+       release: true
+       draft: false
+       prerelease: false
+     versionfilter:
+       kind: semver
+       # pattern accepts any semver constraint
+       pattern: "*"
+
+targets:
+  dockerfile:
+    name: "Bump to latest k3s-root version in Dockerfile"
+    kind: dockerfile
+    scmid: default
+    sourceid: k3sroot
+    spec:
+      file: "Dockerfile"
+      instruction:
+        keyword: "ARG"
+        matcher: "K3S_ROOT_VERSION"
+
+  makefile:
+    name: "Bump to latest k3s-root version in Makefile"
+    kind: file
+    scmid: default
+    disablesourceinput: true
+    spec:
+      file: Makefile
+      matchpattern: '(?m)^K3S_ROOT_VERSION \?\= (.*)'
+      replacepattern: 'K3S_ROOT_VERSION ?= {{ source "k3sroot" }}'
+
+scms:
+  default:
+    kind: github
+    spec:
+      owner: '{{ .scm.default.owner }}'
+      repository: '{{ .scm.default.repository }}'
+      branch: '{{ .scm.default.branch }}'
+      token: '{{ requiredEnv "GITHUB_TOKEN"}}'
+      username: '{{ .scm.default.username }}'
+      user: '{{ requiredEnv "GITHUB_ACTOR"}}'
+      email: '{{ .scm.default.email }}'
```

Comment:
Agree with Olivier's comment. The default values here aren't specified in https://github.com/rancher/image-build-flannel/blob/master/updatecli/values.yaml. You can see an example in the repo's default Updatecli file:

https://github.com/rancher/image-build-flannel/blob/b5f8809a25d349a6ef3ca12d9c4d9ba1584b2285/updatecli/updatecli.d/updatecli.yml#L15-L18

Reply:
Given that they are not in the `values.yaml` and updatecli is supposed to be working, do I need the ones that are not in the values:
```
owner: '{{ .scm.default.owner }}'
repository: '{{ .scm.default.repository }}'
branch: '{{ .scm.default.branch }}'
```
? I wonder if I should add them or drop them from the policy

Reply:
They are specified directly in the current policy file (the only policy available at the moment in this repo): https://github.com/rancher/image-build-flannel/blob/b5f8809a25d349a6ef3ca12d9c4d9ba1584b2285/updatecli/updatecli.d/updatecli.yml#L19-L21

You could migrate those values from `updatecli/updatecli.d/updatecli.yml` to https://github.com/rancher/image-build-flannel/blob/master/updatecli/values.yaml and use them in your policy.

Reply:
Besides this, the rest LGTM.

Reply:
thanks!